pkg/kv/kvserver/consistency_queue.go

```diff
@@ -77,30 +86,50 @@ func newConsistencyQueue(store *Store, gossip *gossip.Gossip) *consistencyQueue
 func (q *consistencyQueue) shouldQueue(
 	ctx context.Context, now hlc.Timestamp, repl *Replica, _ *config.SystemConfig,
 ) (bool, float64) {
-	interval := q.interval()
-	if interval <= 0 {
+	return consistencyQueueShouldQueueImpl(ctx, now,
```

Comment:
Curious why `now` isn't also on `consistencyShouldQueueData`.

Reply:
My mindeset was to wrap everything that we use from Replica in this struct, and now() was part of the original method signature so I left it out. No strong preference on this one, could go either way.